src/main/java/frc/robot/RobotMap.java

```diff
@@ -33,7 +35,11 @@ public static class mapDrivetrain {
     public static final int BACK_RIGHT_ABSOLUTE_ENCODER_CAN = 3;
   }
 
-  public static class mapAlgaeIntake {
-    public static final int ALGAE_MOTOR_CAN = 10;
+  public static class mapClimber {
+    public static final int CLIMBER_CAN = 20;
+
+    public static class mapAlgaeIntake {
+      public static final int ALGAE_MOTOR_CAN = 10;
+    }
   }
```

Comment:
You accidentally made this a subclass by putting it inside the `}`
```suggestion
    }
    public static class mapAlgaeIntake {
      public static final int ALGAE_MOTOR_CAN = 10;
    }
```